integration/constants.go

```diff
@@ -34,6 +34,22 @@ const (
 	DefaultTenGatewayWSPortOffset   = 952
 )
 
+var PortToTestName = map[int]string{
```

Comment:
can we use the actual constants, or maybe reflection?
My fear is that they'll fall out of sync